.github/workflows/deploy.yml

```
@@ -11,10 +11,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.9
+    - name: Set up Python 3.10
       uses: actions/setup-python@v2
       with:
-        python-version: 3.9
+        python-version: "3.10"
```

```suggestion
        python-version: 3.x
```
Force the upgrade to the current Python as soon as it changes.

---

I don't agree on this, we should keep everything on the same version since some steps have to be upgraded manually.

---

It's not a strong opinion, but I'm on the more explicit part with this.

---

@erikwrede Rollbacked the 3.x change, should be okay now.